README.rst

```diff
@@ -7,6 +7,8 @@ audresample
 Python wrapper for `audresamplelib`_
 that provides functions to
 resample and remix audio signals.
+Resampling is only supported for
+single precision signals.
```

Comment:
Maybe we should be a bit more verbose here and change to: " signals in single precision floating-point format."?

Reply:
I changed accordingly.